man/mustache.5.ron

```diff
@@ -484,6 +514,37 @@ Output, assuming the `article.mustache` from before:
     <h1>Yesterday</h1>
     <p>Nothing special happened.</p>
 
+**Dynamic Names**
+
+Some mustache implementations may allow the use of Dynamic Names, an
+**optional** part of the Mustache specification which allows to dynamically
+determine a tag's content at runtime, into parent tags.
+
+Here's an example on how Dynamic Names in parent tags work.
```

Comment:
You can be more concise here, since the optionality of the feature was already mentioned in the previous section.

```suggestion
Some mustache implementations may allow the use of Dynamic Names in
parent tags, similar to dynamic names in partials. Here's an example of
how Dynamic Names in parent tags work.
```

Reply:
Right! That looks nice.